src/components/views/dialogs/security/InitialCryptoSetupDialog.tsx

```diff
@@ -96,4 +97,4 @@ const CreateCrossSigningDialog: React.FC<Props> = ({ matrixClient, accountPasswo
     );
 };
 
-export default CreateCrossSigningDialog;
+export default InitialCryptoSetupDialog;
```

Comment:
Can we change it to a named export too? [According to our code style guide](https://github.com/element-hq/element-web/blob/develop/code_style.md#typescript--javascript)

Reply:
I think we should hold off on over-using named exports for components as it makes React Suspense/lazy much more annoying to use. We should discuss this as a team in a meeting

Reply:
`const FontAwesomeIcon = React.lazy(()=> import('@fortawesome/react-fontawesome').then(module=>({default:module.FontAwesomeIcon})))` is just not as nice as `const Component = React.lazy(()=> import('../component'));`